core/qthelper.cpp

```diff
@@ -1209,11 +1209,19 @@ QString localFilePath(const QString &originalFilename)
 	return localFilenameOf.value(originalFilename, originalFilename);
 }
 
+// TODO: Apparently Qt has no simple way of listing the supported video
+// codecs? Do we have to query them by hand using QMediaPlayer::hasSupport()?
+const QStringList videoExtensionsList = {
+	".avi", ".mp4", ".mpeg", ".mpg", ".wmv"
+};
```

Comment:
the list of extensions looks fine to me, as is.

re: "dodging the bullet"
the point is that we shouldn't care about codecs.

obsolete member to list the MIMEs:
http://doc.qt.io/qt-5/qmediaplayer-obsolete.html
support is only an estimate:
http://doc.qt.io/qt-5/qmultimedia.html#SupportEstimate-enum